internal/db/seeds/dao_test.sql

```diff
@@ -7,4 +7,9 @@ INSERT INTO accounts(id, account_number, org_id)
 VALUES (1, '1', '1')
   ON CONFLICT DO NOTHING;
 
+INSERT INTO pubkeys(id, account_id, name, body)
+VALUES (1, 1, 'lzap-ed25519-2021',
+        'ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIEhnn80ZywmjeBFFOGm+cm+5HUwm62qTVnjKlOdYFLHN lzap')
+ON CONFLICT DO NOTHING;
+
```

Comment:
We need to reset sequences here if we want to add the key.
But do we? Shouldn't we figure out how to use some factories?

Reply:
Oh of course, good catch. While I think static IDs might be useful in tests even more than in dev environment, I will keep what we have today and we can add this later if needed. This could possibly slow down seeding quite a bit when we have like 20 tables in the end.

Reply:
You know what, let's do reset_sequences here and if DAO tests are slow then let's use implicit IDs instead. Since we will use IDs (typically only 1) everywhere, I feel like being explicit is more clean.

Reply:
I'm fine with that :+1: